hoot-core/src/main/cpp/hoot/core/algorithms/zindex/ZCurveRanger.cpp

```diff
@@ -55,12 +55,11 @@ class LongBoxContainer
   LongBoxContainer() {}
 
   LongBoxContainer(LongBox box, long int excess)
+    : _box(box), _excess(excess)
   {
-    _box = box;
-    _excess = excess;
   }
 
-  bool operator=(LongBoxContainer bc) const
+  bool operator==(LongBoxContainer bc) const
```

Comment:
Looking at the return type and the function content this is actually the equality operator and not the addition operator.